java/samples/sample-code/src/main/java/com/microsoft/semantickernel/samples/documentationexamples/SerializingPrompts.java

```diff
@@ -17,29 +18,30 @@ public class SerializingPrompts {
 
     // Only required if AZURE_CLIENT_KEY is set
     private static final String CLIENT_ENDPOINT = System.getenv("CLIENT_ENDPOINT");
-    private static final String MODEL_ID = System.getenv().getOrDefault("MODEL_ID", "gpt-35-turbo-2");
+    private static final String MODEL_ID = System.getenv().getOrDefault("MODEL_ID",
+        "gpt-35-turbo-2");
```

Comment:
gpt-35-turbo-2 is our deployment name, not a model name. I know this is the correct value since the deployment picks the model. But it's confusing because there is no "gpt-35-turbo-2" in the Azure OpenAI service models (at least not that I can see). 

"For Azure OpenAI, the model name must match the deployment name in Azure OpenAI Studio > Deployments." is another thing to add to the readme.